setup.py

```diff
@@ -290,7 +290,12 @@ def run(self):
             'mypy',
             'pytest',
             'requests==2.*',
-            'coverage'
+            'coverage',
+            'pytest-sugar',
```

Comment:
Just curious, I don't see these packages are used anywhere in the code, are they pytest extensions and will be executed automatically?

Reply:
`pytest-sugar` -> If you run pytest locally, this helps to present the tests better

`pytest-cov` -> This changes the "coverage" to use pytest-coverage - better in tracking coverages. The pipeline scripts have been updated to use this. Eg:
```bash
pytest  --instafail --cov=./azure_functions_worker --cov-report xml --cov-branch --cov-append tests/endtoend
```

`pytest-xdist` -> This can run the tests parallel, but the catch currently is that the tests rely on things from other tests (or depend on things which others are also changing at the same time, we need to improve it)

`pytest-randomly` -> This runs the tests randomly - exposes the ordering issues in tests, if any.

`pytest-instafail` -> shows failures and errors instantly instead of waiting until the end of test session, good for running when trying locally.

Reply:
The `pytest-sugar`'s output.

![image](https://user-images.githubusercontent.com/1047040/86058422-ba3f1f80-ba15-11ea-9c7d-be419a19b46e.png)